src/core/useAtom.ts

```diff
@@ -6,7 +6,9 @@ import { Atom, WritableAtom, AnyWritableAtom } from './types'
 
 function assertContextValue<T extends object>(x: T | null): asserts x is T {
```

Comment:
```suggestion
function assertContextValue<T extends object>(x: T | null, scope?: unknown): asserts x is T {
```

Let's show the note about the scope only if scope is specified. I suppose, the majority of users don't use scopes.